### PR TITLE
Improvements to CLI messaging

### PR DESCRIPTION
**Pull request by ssk2**
These are some improvements to CLI messaging that were suggested by @keithchambers last week.

### CODE REVIEW

File: cli/commands/update.go [hunk 1]
```diff
@@ -97,6 +98,7 @@ func parseUpdateResponse(responseBytes []byte) (string, error) {
 }
 
 func doUpdate(optionsFile, packageVersion string) {
+	config.Command = "update"
```
Comment (benclarkwood) [inline cli/commands/update.go]: This seems like it has weird side effects. Config is thrown away after the command completes?

Comment (ssk2) [inline cli/commands/update.go]: @benclarkwood yup - it's in memory. I'll move this to the actual handler method instead so it's only ever set upon invocations of that command.

**Comment by ssk2** [comment]
@benclarkwood: amended. I removed a couple of fields that weren't being used, and used the actual command being passed through by Kingpin. `config.Command` is now set in the callback methods for `describe` and `handle` respectively.

**Comment by ssk2** [comment]
retest this please

**Pull request merged**